recipes/gfapy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: gfapy
+  version: "1.0.0rc9"
+
+source:
+  fn: gfapy-1.0.0rc9.tar.gz
+  url: https://pypi.python.org/packages/45/ac/e0cb4068c34fa81a8c1446b96e36f536d66984047fa9fffdd8d88b84f919/gfapy-1.0.0rc9.tar.gz
+  md5: 155eeb5ff28fdde596129baef3cd9529
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
+  imports:
+    - gfapy
+    - gfapy.alignment
+    - gfapy.field
+    - gfapy.graph_operations
+    - gfapy.line
+    - gfapy.line.comment
+    - gfapy.line.common
+    - gfapy.line.custom_record
+    - gfapy.line.edge
+    - gfapy.line.edge.common
+    - gfapy.line.edge.containment
+    - gfapy.line.edge.gfa1
+    - gfapy.line.edge.gfa2
+    - gfapy.line.edge.link
+    - gfapy.line.fragment
+    - gfapy.line.gap
+    - gfapy.line.group
+    - gfapy.line.group.gfa2
+    - gfapy.line.group.ordered
+    - gfapy.line.group.path
+    - gfapy.line.group.unordered
+    - gfapy.line.header
+    - gfapy.line.segment
+    - gfapy.line.unknown
+    - gfapy.lines
+    - tests
+
+  requires:
+    - nose
+
+about:
+  home: https://github.com/ggonnella/gfapy
+  license: ISC License (ISCL)
+  summary: 'Library for handling data in the GFA1 and GFA2 formats'
+  license_family: OTHER
```

Comment:
Looks like the Travis build is failing at least in part due to the `license_family`. Try changing it from `OTHER` to `Other`.